<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\bin\Debug\ReportCardGenerator.exe" total="1" errors="1" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2010-07-01" time="14:03:24">
  <environment nunit-version="2.5.5.10112" clr-version="2.0.50727.1433" os-version="Microsoft Windows NT 5.1.2600 Service Pack 2" platform="Win32NT" cwd="C:\Program Files\NUnit 2.5.5\bin\net-2.0" machine-name="MANA-BA3F9C25D8" user="Jun" user-domain="MANA-BA3F9C25D8" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\bin\Debug\ReportCardGenerator.exe" executed="True" result="Failure" success="False" time="0.000" asserts="0">
    <results>
      <test-suite type="Namespace" name="ReportCardGenerator" executed="True" result="Failure" success="False" time="0.000" asserts="0">
        <results>
          <test-suite type="Namespace" name="Tests" executed="True" result="Failure" success="False" time="0.000" asserts="0">
            <results>
              <test-suite type="TestFixture" name="ControllerTest" executed="True" result="Failure" success="False" time="0.000" asserts="0">
                <results>
                  <test-case name="ReportCardGenerator.Tests.ControllerTest.testRemoveStudent" executed="True" result="Error" success="False" time="0.000" asserts="0">
                    <failure>
                      <message><![CDATA[System.ArgumentOutOfRangeException : Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index]]></message>
                      <stack-trace><![CDATA[at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
at System.ThrowHelper.ThrowArgumentOutOfRangeException()
at System.Collections.Generic.List`1.RemoveAt(Int32 index)
at ReportCardGenerator.Controller.StudentController.removeStudent(Student stud) in C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\Controller\StudentController.cs:line 145
at ReportCardGenerator.Controller.StudentController.removeStudent(String studentID) in C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\Controller\StudentController.cs:line 154
at ReportCardGenerator.Tests.ControllerTest.testRemoveStudent() in C:\Documents and Settings\Jun\My Documents\Southville project\ReportCardGenerator\ReportCardGenerator\Tests\ControllerTest.cs:line 82
]]></stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>